base/common/src/main/java/com/netscape/cmsutil/crypto/CryptoUtil.java

```diff
@@ -254,290 +256,61 @@ public class CryptoUtil {
-            {
-                add("c2pnb176v1");
-            }
-        });
-        ecOIDs.put("1.2.840.10045.3.0.5", new Vector<String>() {
-            {
-                add("c2tnb191v1");
-            }
-        });
-        ecOIDs.put("1.2.840.10045.3.0.6", new Vector<String>() {
-            {
-                add("c2tnb191v2");
-            }
-        });
-        ecOIDs.put("1.2.840.10045.3.0.7", new Vector<String>() {
-            {
-                add("c2tnb191v3");
-            }
-        });
-        ecOIDs.put("1.2.840.10045.3.0.10", new Vector<String>() {
-            {
-                add("c2pnb208w1");
-            }
-        });
-        ecOIDs.put("1.2.840.10045.3.0.11", new Vector<String>() {
-            {
-                add("c2tnb239v1");
-            }
-        });
-        ecOIDs.put("1.2.840.10045.3.0.12", new Vector<String>() {
-            {
-                add("c2tnb239v2");
-            }
-        });
-        ecOIDs.put("1.2.840.10045.3.0.13", new Vector<String>() {
-            {
-                add("c2tnb239v3");
-            }
-        });
-        ecOIDs.put("1.2.840.10045.3.0.16", new Vector<String>() {
-            {
-                add("c2pnb272w1");
-            }
-        });
-        ecOIDs.put("1.2.840.10045.3.0.17", new Vector<String>() {
-            {
-                add("c2pnb304w1");
-            }
-        });
-        ecOIDs.put("1.2.840.10045.3.0.19", new Vector<String>() {
-            {
-                add("c2pnb368w1");
-            }
-        });
-        ecOIDs.put("1.2.840.10045.3.0.20", new Vector<String>() {
-            {
-                add("c2tnb431r1");
-            }
-        });
-        ecOIDs.put("1.3.132.0.6", new Vector<String>() {
-            {
-                add("secp112r1");
-            }
-        });
-        ecOIDs.put("1.3.132.0.7", new Vector<String>() {
-            {
-                add("secp112r2");
-            }
-        });
-        ecOIDs.put("1.3.132.0.28", new Vector<String>() {
-            {
-                add("secp128r1");
-            }
-        });
-        ecOIDs.put("1.3.132.0.29", new Vector<String>() {
-            {
-                add("secp128r2");
-            }
-        });
-        ecOIDs.put("1.3.132.0.4", new Vector<String>() {
-            {
-                add("sect113r1");
-            }
-        });
-        ecOIDs.put("1.3.132.0.5", new Vector<String>() {
-            {
-                add("sect113r2");
-            }
-        });
-        ecOIDs.put("1.3.132.0.22", new Vector<String>() {
-            {
-                add("sect131r1");
-            }
-        });
-        ecOIDs.put("1.3.132.0.23", new Vector<String>() {
-            {
-                add("sect131r2");
-            }
-        });
-    }
+    public static final Map<String, Vector<String>> ecOIDs = Map.ofEntries(
+            Map.entry("1.2.840.10045.3.1.7", new Vector<>(List.of("nistp256", "secp256r1"))),
```

Comment:
Nice! In the future we might be able to convert this field into a `Map<String, List>` to make it even simpler.